clients/python/sust/api/climate_explorer.py

```diff
@@ -176,6 +180,11 @@ def __next__(self):
             resp = self.req_func(*self.req_args, **self.req_kwargs)
             self.cache = resp
 
+            import logging
```

Comment:
This little guy snuck in here.

Reply:
I'm always good to accidentally leave at least one random logging statement in 😆